src/labels/CustomLabels.labels

```diff
@@ -5085,6 +5085,13 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>Button to process a batch.</shortDescription>
         <value>Process Batch</value>
     </labels>
+    <labels>
```

Comment:
@melissabarber - I've added these two labels to update the "Process Batch" button (top label) and to add a new block of text to the Elevate widget in batch mode (bottom label). Full details of these labels can be found in the AC here, https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B00000094E17IAE/view.

Do these labels and their names work or should we make any adjustments here?